### PR TITLE
Add homepage

### PR DESCRIPTION
**Pull request by MesakDuduCoder**
This is my Capstone project for module one, it is an online learning website. It has cool animation effects and a stylish and modern design. I hope that you like it and take something away from my work.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                    To have your course featured in our platform it needs to meet certain criteria, you have to take a look at our program for creating a course and make sure you follow the requirements after that your course will be reviewed by our team of reviews then if it goes well it will be approved and featured here at TOP CLASS.
+                </h2>
+                <div id="aboutPage-logo" class="d-flex justify-content-center align-items-center p-3 mb-4">
+                    <img src="./icons/notebook-512.png" alt="home-icon" width="120" height="80">TOP&nbsp;CLASS
+                </div>
+            </div>
+            <div id="separator" class="container-fluid"></div>
+            <div class="container-fluid d-flex flex-column align-items-center h-50 m-5 p-4">
+                <h1 id="aboutPage-sub-title-3">Check out our best selling courses</h1>
+                <div class="orange-line-2 mb-3 mt-1"></div>
```
Comment (m05e5) [inline about.html:124]: - [ ] Hey. You are required to follow HTML semantics. So, kindly make sure you have only one `h1` tag per page

File: about.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                <h2 id="aboutPage-para-3">Our React for Beginners Course by John Kim and Python for Beginners by Edin Diaz.</h2>
+                <div class="m-md-4">
+                    <img class="best-selling-img p-3" src="./images/python.webp" alt="">
+                    <img class="best-selling-img p-3" src="./images/react.webp" alt="">
+                </div>
```
Comment (m05e5) [inline about.html:129]: - [ ] Please as part of the requirements you are supposed to follow the design as much as possible. Kindly make sure that for this section, the images are transparent with an orange background color and some text on it as it is on the design

**Design**
image

**Your app**
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    <section id="siteHome" class="container-fluid p-3 p-md-5 m-0">
+        <div class="p-md-5 m-md-5 p-3 m-1 d-flex flex-column flex-md-wrap">
+            <p id="welcome-text"class="py-1">"Hello! Welcome to TOP CLASS"</p>
+            <p id="main-title" class="py-3 m-0">LEVEL UP YOUR SKILLS <br> WITH US</p>
+            <div class="main-title-sub bg-light p-md-3 p-1 mb-4">
+                <p id="main-para" class="py-1">Join any course from the selection of many free and paid courses</p>
+            </div>
+            <p id="main-sub-title" class="py-1">COME MAKE LEARNING FUN WITH US</p>
+        </div>
+    </section>
```
Comment (m05e5) [inline index.html:87]: - [ ] You are required to follow HTML semantics. So, kindly make sure you have at least one `h1` tag per page

File: index.html [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                <h2 id="sponcer-title">Partner</h2>
+                <div class="orange-line-2 mb-3 mt-1"></div>
+                <div class="partners-img-box">
+                    <img class="sponcer-images" src="./images/mozilla.png" alt="">
+                    <img class="sponcer-images" src="./images/google.png" alt="">
+                    <img class="sponcer-images" src="./images/hyundai.png" alt="">
+                    <img class="sponcer-images" src="./images/uber.png" alt="">
+                    <img class="sponcer-images" src="./images/air-bnb.png" alt="">
+                </div>
+            </div>
```
Comment (m05e5) [inline index.html:122]: - [ ] For accessibility purpose and to make sure you follow HTML semantics please make sure you have a significant alternative text for each image.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+                <div class="d-flex flex-column justify-content-center py-3 pe-md-5 pe-1">
+                    <h2 class="cc-text">2015 Creative Commons. Some Rights Reserved</h2>
+                    <h2 class="cc-text d-none d-md-block">
+                        Creative Commons is an American non-profit organization and international network devoted to educational access
+                        and
+                        expanding the range of creative works available for others to build upon legally and to share.
+                    </h2>
+                </div>
+            </div>
+        </div>
```
Comment (m05e5) [inline about.html:146]: - [ ] You are required to follow HTML semantics. Kindly make sure every page has a `footer` tag

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @MesakDuduCoder , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders



## Highlights

✔️ Very nice UI
✔️ No linters error
✔️ Professional readme file

## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MesakDuduCoder** [comment]
Can you please request further changes as optional because I'm running out of time and I don't have any block repeat left.

**Review by ABDULALI3468 — APPROVED** [review]
Hi @MesakDuduCoder , 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**